kele.el

```diff
@@ -341,26 +341,22 @@ retval into `async-wait'."
   "Start file-watch for CACHE.
 
 If BOOTSTRAP is non-nil, perform an initial read."
-  (oset cache
-        filewatch-id
-        (kele--fnr-add-watch
-         (f-join kele-cache-dir "discovery/")
-         '(change)
-         (-partial #'kele--cache-update cache)))
-  (when bootstrap
-    (kele--cache-update cache)))
+  (oset cache timer
+        (run-with-timer
+         (if bootstrap 0 kele-discovery-refresh-interval)
+         kele-discovery-refresh-interval
```

Comment:
Document:
- How to change the refresh interval;
- The nuance that the refresh interval is implicitly reliant on `kubectl`'s 10-minute-ish cache updating cycle as well, which means that the update here may not "actually" result in the truly up-to-date discovery cache in Kele (worth ticketing for follow-up);
- A TLDR explanation of why we do this in timer-based fashion and link to `kele-01` for further details.